contracts/OriginationController.sol

```diff
@@ -949,7 +962,7 @@ contract OriginationController is
         }
 
         // approve LoanCore to take the total settled amount
-        payableCurrency.approve(address(loanCore), settledAmount);
+        payableCurrency.safeApprove(address(loanCore), settledAmount);
```

Comment:
Would it make sense to make this a transfer function similar to the loan origination flow where the funds are sent straight to LC? Seems like the pattern used here should match the 'start loan' flow.

Reply:
Nevermind I think I can answer my own question. Thinking more about this, safer to do a pull over a push of funds. Especially since rollovers are a technically a repayment flow. So no change necessary imo, I like it.